README.md

```diff
@@ -28,3 +28,6 @@ The purpose of this repository is to provide a Clang-Format configuration for Un
 
 **Optional:** You can enable `format on save` by installing [this extension](https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave).
 
+## Limitations
```

Comment:
Can move or delete this depending on how you want your readme. These are pretty big limitations though that we actively work around (for instance, we flat out turn off clang-format around Slate lines).